conf/machine/include/rpi-base.inc

```diff
@@ -2,6 +2,10 @@ include conf/machine/include/rpi-default-settings.inc
 include conf/machine/include/rpi-default-versions.inc
 include conf/machine/include/rpi-default-providers.inc
 
+require conf/multilib.conf
+MULTILIBS = "multilib:lib32"
+DEFAULTTUNE_virtclass-multilib-lib32 = "armv7athf-neon-vfpv4"
+
```

Comment:
This bit doesn't look like a backport - I can't see it in the newer branches.

Reply:
@pbrkr This is used for supporting 32bit lib on 64bit platform. When some 3rd party binary is delivered on a 32bit form, we can still run it on raspberrypi3-64 version.

Reply:
We shouldn't be adding features to old branches which haven't been merged to master as it fragments our support. It'd be good to see this proposed for master though.

Reply:
yes enabling multilib is a distro choice not BSP choice. Please remove this

Reply:
@kraj OK, I will remove this snippets